src/app/helptext/system/alert-settings.ts

```diff
@@ -0,0 +1,276 @@
+import { T } from '../../translate-marker';
+
+export default {
```

Comment:
Does each type of alert need a separate tooltip, or can we create a one help text field for "level" and one for "policy" and have the pages/system/alert.component.ts file use these two tooltips for each "Warning Level" and "Set Frequency" field that is created?

Reply:
@Mrt134 I've already discussed with Lola that it's going to be 2 tool tips total in source. They will be applied to each clone in the UI. Working on this now.